addons/website_payment_paypal/views/res_config_settings_views.xml

```diff
@@ -1,12 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
+<!--NO LONGER USED-->
 <odoo>
     <record id="res_config_settings_view_form" model="ir.ui.view">
         <field name="name">res.config.settings.view.form.inherit.website.payment.paypal</field>
         <field name="model">res.config.settings</field>
         <field name="priority" eval="20"/>
         <field name="inherit_id" ref="website_payment.res_config_settings_view_form"/>
         <field name="arch" type="xml">
-            <div id="website_payment_right_pane" position="inside">
```

Comment:
Woopsie :see_no_evil: 

![image](https://user-images.githubusercontent.com/15896813/220310363-49e15731-b9de-49d5-ad56-1e2b13ffa11e.png)

Reply:
@AntoineVDV  How did you manage to do that :O I am not able to recreate it.